lib/Service/SynchronizationService.php

```diff
@@ -2266,14 +2264,15 @@ private function processWriteFileRule(Rule $rule, array $data, string $objectId,
             $openRegisters->setSchema($schemaId);
 
             try {
-                // Write file with OpenRegister ObjectService.
+                $tags = array_merge($config['tags'] ?? [], ["object:$objectId"]);
+                
                 $objectService = $this->containerInterface->get('OCA\OpenRegister\Service\ObjectService');
-                $file = $objectService->addFile(object: $objectId, fileName: $fileName, base64Content: $content);
+                $objectEntity = $objectService->findByUuid(uuid: $objectId);
+
+                // Write file with OpenRegister FileService.
+                $fileService = $this->containerInterface->get('OCA\OpenRegister\Service\FileService');
```

Comment:
What if OpenConnector runs without OpenRegister installed. Shouldn't we catch an exception here?